example/deploy/crds/mellanox.com_v1alpha1_nicclusterpolicy_cr.yaml

```diff
@@ -36,6 +36,20 @@ spec:
           }
         ]
       }
+  secondaryNetwork:
+    cniPlugins:
+      image: containernetworking-plugins
```

Comment:
We should have a dockerfile that builds this.

Unsure if it should be part of the project or maintained in another repo